src/app/modules/group/components/group-invite-users/group-invite-users.component.ts

```diff
@@ -1,24 +1,166 @@
-import { Component, Input, Output, OnInit, EventEmitter } from '@angular/core';
+import { Component, Input, Output, OnInit, EventEmitter, ViewChild } from '@angular/core';
+import { TextareaComponent } from 'src/app/modules/shared-components/components/textarea/textarea.component';
+import { MessageService } from 'primeng/api';
+import { ERROR_MESSAGE } from 'src/app/shared/constants/api';
+import { TOAST_LENGTH } from 'src/app/shared/constants/global';
+import { CreateGroupInvitationsService, InvitationResult } from '../../http-services/create-group-invitations.service';
 import { Group } from '../../http-services/get-group-by-id.service';
 
+interface Message
+{
+  type: 'success' | 'info' | 'error',
+  summary?:string,
+  detail:string
+}
+
 @Component({
   selector: 'alg-group-invite-users',
   templateUrl: './group-invite-users.component.html',
-  styleUrls: ['./group-invite-users.component.scss']
+  styleUrls: ['./group-invite-users.component.scss'],
+  providers: [MessageService],
 })
 export class GroupInviteUsersComponent implements OnInit {
 
   @Input() group: Group
   @Output() refreshRequired = new EventEmitter<void>();
 
-  showMsg = false;
+  @ViewChild(TextareaComponent)
+  private textArea: TextareaComponent;
+
+  logins:string[];
+
+  processing = false;
 
-  constructor() { }
+  tooManyLogins = false;
+
+  messages : Message[] = [];
+
+  canClick:boolean;
+
+  constructor(
+    private createGroupInvitationsService: CreateGroupInvitationsService,
+    private messageService: MessageService,
+    ) {}
 
   ngOnInit(): void {}
```

Comment:
remove it if you do not use it

Reply:
the `messageService` is used when the `createGroupInvitationsService` returns an error (an error toast is displayed)

Reply:
My remark was on the `ngOnInit(): void {}` line. If this function is not used, no need to let it there.